src/client/components/DashbordItems/Dasbbord.stories.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import DashbordItems from './Dashbord.component';
+import DashbordItems from './dashbord.component';
```

Comment:
I think the folder name should be dashboard-items, and then inside files 
dashboard-items.component.js
dashboard-items.stories.js
dashboard-items.style.css
I couldn't start storybook for example with this change.